doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -362,6 +362,11 @@ nRF Desktop
     This is done to work around the timeout in MPSL flash synchronization (``NCSDK-29354`` known issue).
     See :ref:`known_issues` for details.
 
+* Added DVFS module to the nRF Desktop application that can be used on the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+  The module is responsible for switching the frequency and voltage according to the application's needs.
+  The module ensures that lowest DVFS frequency and voltage is used unless computation-challenging operation requests higher frequency and voltage.
+  Module listens to Event Manager events and applies DVFS frequency according to the application state.
```

Comment:
Added this suggestion to PR https://github.com/nrfconnect/sdk-nrf/pull/18192/files. Please remove this file. 
The ref link of DVFS module will be added later after this PR is merge to avoid build failure in the release notes PR.